stackbrew.js

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
```

Comment:
I'm not sure what the benefit would be, since it's really a utility script that isn't shareable outside this repo. If you want to take a stab after, feel free, but I don't really want to spend much more time on this work (now that is is running)